src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Razor/TextReplacement.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.DotNet.UpgradeAssistant.Steps.Razor
+{
+    public class TextReplacement : IEquatable<TextReplacement>
```

Comment:
based on the PR i think the reason we use this class is to store a list of replacements, without performing the replacement, as we analyze because replacing would change the location of n+1 diagnostic.

seems like a comment on the class could help clarify why it was necessary. when i first saw it i wondered why we didn't just do the replacement immediately.

Reply:
I'll add a comment.